scripts/partition_manager_report.py

```diff
@@ -30,7 +30,11 @@ def print_region(domain, region, size, pm_config):
         bcolors = bcolors_ansi
 
     # Print header
-    print(bcolors["OKBLUE"] + "%s %s (0x%x - %dkB):" % (domain, region, size, size/1024) + bcolors["ENDC"])
+    # print(bcolors["OKBLUE"] + "%s %s (0x%x - %dkB):" % (region, size, size / 1024) + bcolors["ENDC"])
+    if size >= 1024:
+        print(f"{bcolors['OKBLUE']} {domain} {region} (0x{size:x} - {int(size/1024):d}kB): {bcolors['ENDC']}")
```

Comment:
Could you pull out the difference between these two prints and assign the value to a variable which you pass to a single print expression?

```
val = f'{int(size/1024):d}kB' if size >= 1024 else f'{size:d}B'
print(... val ...)
```